apps/confidential/confidential_test.go

```diff
@@ -164,6 +164,76 @@ func TestAcquireTokenByCredential(t *testing.T) {
 	}
 }
 
+func TestRegionAutoEnable(t *testing.T) {
```

Comment:
I would like to see all 3 acceptance tests independently, as this is difficult to read.

Reply:
Updated!